src/codegen/tests/snapshots/rusty__codegen__tests__expression_tests__int_to_int_expt.snap.new

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
i think we can remove these snapshots ?

Reply:
> you got two snapshots that sneaked in snap.new

they successfully avoided the *.snap search :)